src/js/components/Form/Form.js

```diff
@@ -455,6 +481,12 @@ const Form = forwardRef(
                 setTouched(nextTouched);
               }
 
+              if (!currentlySelectedElement[name]) {
+                setCurrentlySelectedElement({
```

Comment:
I'm not really seeing how this is any different than `touched`. It seems to mirror it. Why the separate flag?

Reply:
Yes, you are right. We don't need that, while iterating through different approaches to tackle this, I had added it then. But, I will remove it.